compiler/rustc_hir_analysis/src/outlives/utils.rs

```diff
@@ -144,7 +144,7 @@ pub(crate) fn insert_outlives_predicate<'tcx>(
     }
 }
 
-fn is_free_region(region: Region<'_>) -> bool {
+fn is_early_bound_region(region: Region<'_>) -> bool {
```

Comment:
Just a drive-by renaming, cc #117908. I don't know why this used the term *free* (`ReLateParam`). It should've always be named `is_early_bound_region` or sth. in that vein.